### ISSUE TITLE
Novatel SPAN PwrPak7 and IMU-IGM-S1 report bad location when vehicle is moving on Lexus; JIRA Issue: #CAR-869

### ISSUE THREAD

**Issue by msmcconnell** [usdot-fhwa-stol/CARMAPlatform#275]
### Types of Issue

- [x] Anomaly report (something appears to not work correctly)
- [ ] Enhancement request (describe the enhancement being requested)
- [ ] Other (please ensure the description clarifies why the issue doesn’t fall into either of the above categories)

### Descriptive summary
It has been observed that when the blue lexus at TFHRC is in motion the fix reported by the Novatel GPS/IMU can experience large amounts of drift. When the vehicle is stationary however, the reported fix is very accurate. It is not clear at the moment what the root cause is. Some possibilities include bad IMU/GPS configuration or bad hardware. 
### Expected behavior
Minimal drift when good GNSS fix is available and vehicle is moving slowely
### Actual behavior
Lots of drift can be observed
### Steps to reproduce the behavior
Drive the car while observing reported GNSS locations

**Comment by msmcconnell** [comment]
![image](https://user-images.githubusercontent.com/10292079/60592851-76b4c580-9d6f-11e9-8748-9e88e28b19fa.png)
Above shows the reported location of the vehicle while driving a small loop. Purple is the actual path. Red is the BESTPOS result.
Data collected using the following configuration
```
CONNECTIMU COM2 IMU_STIM300
SETINSTRANSLATION ANT1 -0.62 0.138 1.20925 0.05 0.05 0.05
SETINSTRANSLATION ANT2  0.555 0.138 1.20925 0.05 0.05 0.05
SETINSTRANSLATION USER -0.036 -0.01 -0.2566
SETINSROTATION RBV 0.0 0.0 0.0 3.0 3.0 3.0
HEADINGOFFSET -90.0 0.0
PPSCONTROL ENABLE POSITIVE 0.5 2000
LOG COM3 GPRMC ONTIME 1 0.5
ETHCONFIG ETHA AUTO AUTO AUTO AUTO
ICOMCONFIG ICOM1 TCP :2000
IPCONFIG ETHA STATIC 192.168.74.10 255.255.255.0 192.168.74.1
SAVEETHERNETDATA ETHA
SAVECONFIG
```

Per the novatel documentation this configuration was updated to 
```
CONNECTIMU COM2 IMU_STIM300
SETINSTRANSLATION ANT1 -0.62 0.138 1.20925 0.05 0.05 0.05
SETINSTRANSLATION ANT2  0.555 0.138 1.20925 0.05 0.05 0.05
SETINSTRANSLATION USER -0.036 -0.01 -0.2566
SETINSROTATION USER 0.0 0.0 0.0 3.0 3.0 3.0
SETINSROTATION RBV 0.0 0.0 0.0 3.0 3.0 3.0
SETINSPROFILE LAND_BASIC
HEADINGOFFSET -90.0 0.0
PPSCONTROL ENABLE POSITIVE 0.5 2000
LOG COM3 GPRMC ONTIME 1 0.5
ETHCONFIG ETHA AUTO AUTO AUTO AUTO
ICOMCONFIG ICOM1 TCP :2000
IPCONFIG ETHA STATIC 192.168.74.10 255.255.255.0 192.168.74.1
SAVEETHERNETDATA ETHA
SAVECONFIG
```
This resulted in the following run
![image](https://user-images.githubusercontent.com/10292079/60593111-12463600-9d70-11e9-8b0b-041fbcdb8cf5.png)
This is an improvement but by no means sufficient.

**Comment by msmcconnell** [comment]
Resolved by PR #334

It was discovered that the HEADINOFFSET setting on novatel cannot be used in conjunction with SPAN. It causes bad initialization of the INS solution. Therefore, the GPS api was reconfigured to use gps_common/GPSFix messages and the novatel driver updated to publish that data from the INSPVAX log.
